graph-proxy/src/graphql/workflows.rs

```diff
@@ -294,19 +294,7 @@ struct Artifact<'a> {
 impl Artifact<'_> {
     /// The file name of the artifact
     async fn name(&self) -> Result<&str, WorkflowParsingError> {
-        let path = Path::new(
-            self.manifest
-                .s3
-                .as_ref()
-                .ok_or(WorkflowParsingError::UnrecognisedArtifactStore)?
-                .key
-                .as_ref()
-                .ok_or(WorkflowParsingError::MissingArtifactKey)?,
-        );
-        path.file_name()
-            .unwrap_or_default()
-            .to_str()
-            .ok_or(WorkflowParsingError::InvalidArtifactFilename)
+        get_artifact_filename(self.manifest)
```

Comment:
The behaviour here appears to change, now we return the entire field as a `&str` where before we took the `path.file_name()` segment